endpoints/openrtb2/auction.go

```diff
@@ -362,6 +366,11 @@ func validateBidAdjustmentFactors(adjustmentFactors map[string]float64, aliases
 	return nil
 }
 
+func validateSChains(req *openrtb_ext.ExtRequest) error {
+	_, err := exchange.BidderToPrebidSChains(req)
```

Comment:
Do we want a simplified version that just checks for duplicate bidders, or perhaps save the result from here for use later?